ProjectImNotSureWhatItIS/User.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ProjectImNotSureWhatItIS
+{
+    class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string FavAnimal { get; set; }
+        public string FavColor { get; set; }
+        public int FavNumber { get; set; }
+        public int NumPets { get; set; }
+        public string petNames { get; set; }
+        public string AnswerQuestions(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+        public int AnswerQuestionWithIntiger(string question)
+        {
+            string stringAnswer = AnswerQuestions(question);
+            return int.Parse(stringAnswer);
+        }
+        public int AnswerQuestionWithIntiger2(string question)
```

Comment:
This method could have been left out, mainly because your `AnswerQuestionsWithIntiger` method could have been used more than once. I was also a bit confused with the `int x = int.Parse(stringAnswer)`...then `return int.Parse(stringAnswer)`

Reply:
that line came off repetitive